tests/PHPStan/Analyser/AnalyserTraitsIntegrationTest.php

```diff
@@ -14,7 +14,9 @@ class AnalyserTraitsIntegrationTest extends \PHPStan\Testing\TestCase
 
 	protected function setUp(): void
 	{
-		$this->fileHelper = $this->getContainer()->getByType(FileHelper::class);
+		parent::setUp();
```

Comment:
Using `parent::setUp()` everywhere won't be necessary if you implement my suggestion.